rime_engine.c

```diff
@@ -308,6 +308,7 @@ static void ibus_rime_engine_update(IBusRimeEngine *rime_engine)
   gboolean inline_preedit =
       g_ibus_rime_settings.embed_preedit_text && context.commit_text_preview;
   gboolean highlighting =
+      (g_ibus_rime_settings.color_scheme->color_scheme_id != NULL) &&
```

Comment:
you change made this always NULL and disabled the preedit color scheme feature entirely. I guess the following code path for `highlighting == true` case is not tested.

Reply:
Yes, I haven't tested that.